lib/scold/hound.yml

```diff
@@ -126,13 +127,23 @@ Style/TrailingCommaInArguments:
   SupportedStylesForMultiline:
   - comma
   - no_comma
-Style/TrailingCommaInLiteral:
-  Description: Checks for trailing comma in literals.
+Style/TrailingCommaInArrayLiteral:
+  Description: Checks for trailing comma in array literals.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas
   Enabled: false
   EnforcedStyleForMultiline: no_comma
   SupportedStylesForMultiline:
   - comma
+  - consistent_comma
+  - no_comma
+Style/TrailingCommaInHashLiteral:
+  Description: Checks for trailing comma in hash literals.
+  StyleGuide: https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas
+  Enabled: false
```

Comment:
Isn't this disabling the check?

I thought the rails team was operating with a "no trailing comma" rule. The frontend guild expects trailing commas everywhere.

Reply:
It is, but that's the existing behavior.